requirements.txt

```diff
@@ -12,3 +12,4 @@ h5py
 requests
 shapely
 tqdm
+conda
```

Comment:
Can you explain why we need conda as a requirement in a file that is installed with conda?(I'm not following)

Reply:
One of the bits of metadata that can be included in the archive is the specification for the conda environment that ran the simulation. To get the envirenment, the python code uses the python api to conda, and so the conda package proper must be present.

Reply:
You mean you are in a situation where this code is not running in a conda environment but needs to get information from a conda environment elsewhere?

Reply:
No, the code is running in a conda environment, and we want to record the details of that environment as metadata along with the results of a simulation run using code in that environment, to make it easy to answer questions like "What version of X was being using in the process that produced output Y?"

Reply:
So in that case conda is already installed in your conda env and you don't need pypi to install it. I don't think it should be listed as a formal dependency but should have an import protection around it like we do in lsst-utils package when retrieving conda package versions.

Reply:
Yes. That's why we add `conda` to `rubin-env`. Does adding it to the formal dependencies cause problems when pip installing?

Reply:
Since we have a pyproject.toml file with project dependencies, I *think* you shouldn't have a problem with pip installation. I guess if conda is part of Rubin-env, it's reasonable to add as a dependency for this package.